### PR TITLE
feat: Undo/redo documentation

### PR DESCRIPTION
**Pull request by sydopy**
## Overview
Provide documentation for undo/redo functionality

## Tests
[Include a link to the passing GitHub action running the test suite here.]

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: docs/UndoRedo.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+## Properties
+
+### `State`
+
+The dashboard state stores all necessary information to render dashboard. 
+
+Type: Object
+
+A state contains the following fields:
```
Comment (tjuranek) [inline docs/UndoRedo.md:13]: Probably don't need the whole dashboard state fields documented here, only the ones related to Undo/Redo. `stretchToFit` seems like it'd be unrelated as well as a couple others.

File: docs/UndoRedo.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    width: DEFAULT_WIDTH,
+    cellSize: DEFAULT_CELL_SIZE,
+    intermediateDashboardConfiguration: undefined,
+    undoQueue: [onMoveAction({position: {x: 1,y:1}, isActionComplete: True, dashboardConfiguration})],
+    redoQueue: [],
+    previousPosition: undefined,
+  };
+```
+
+### `Actions`
```
Comment (tjuranek) [inline docs/UndoRedo.md:98]: What are all the actions that are responsible for undoing and redoing? They could be documented here.

**Review by tjuranek — APPROVED** [review]

**Comment by tracfren** [comment]
Can this be merged?

**Comment by tjuranek** [comment]
> Can this be merged?

Merging to avoid the PR becoming stale, comments can be addressed at a later time.

**Pull request merged**